tests/cli_test.py

```diff
@@ -0,0 +1,56 @@
+import sys
+from tests.draw_test import TestBase
+import unittest
+from worldengine import __main__
+from worldengine.cli.main import main
+
+
+class TestCLI(TestBase):
+
+    def setUp(self):
+        super(TestCLI, self).setUp()
+        self.world = "%s/seed_28070.world" % self.tests_data_dir
+
+    def test__main__(self):
```

Comment:
yes, the two underscore have this special meaning (just by convention) when present at the start of a name, but I thought it could be confusing to have them at the end (someone could wonder if they have the same meaning of the underscores at the start). But it was just over-worrying, let's drop my comment